prov/efa/src/efa_mr.c

```diff
@@ -374,10 +374,6 @@ static int efa_mr_cache_regattr(struct fid *fid, const struct fi_mr_attr *attr,
 	efa_mr = (struct efa_mr *)entry->data;
 	efa_mr->entry = entry;
 
-	ret = efa_mr_hmem_setup(efa_mr, attr);
```

Comment:
So it's because `ofi_mr_cache_search` call `add_region`, which is `efa_mr_reg_impl`, which calls `efa_mr_hmem_setup`. Right?

Reply:
Yes.